### PR TITLE
Image service etag

### LINKED ISSUE #210: Refine caching for image-service
- Changing templates should bust the cache
  - Maybe using an `etag` on the hash of the content or the modification date of the template

### PR DESCRIPTION
**Pull request by icepuma**
Closes #210

### CODE REVIEW

File: projects/rawkode-academy/image-service/package.json [hunk 1]
```diff
@@ -3,18 +3,18 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "prebuild": "cp node_modules/@resvg/resvg-wasm/index_bg.wasm src/wasm/resvg.wasm && cp node_modules/yoga-wasm-web/dist/yoga.wasm src/wasm/yoga.wasm",
-    "prestart": "cp node_modules/@resvg/resvg-wasm/index_bg.wasm src/wasm/resvg.wasm && cp node_modules/yoga-wasm-web/dist/yoga.wasm src/wasm/yoga.wasm",
+    "prebuild": "./scripts/copy_wasm.sh",
+    "prestart": "./scripts/copy_wasm.sh",
```
Comment (rawkode) [inline projects/rawkode-academy/image-service/package.json:7]: Maybe instead of a scripts directory we should use a Justfile?

**Comment by rawkode** [comment]
A hash of the template was my thinking too 😄

File: projects/rawkode-academy/image-service/package.json (hunk 1 above)
Comment (icepuma) [reply projects/rawkode-academy/image-service/package.json:7]: Justfile wouldn't work. I want to run this step every time a `start` or `build` happens and Justfile doesn't work on a Cloudflare build, right?

Comment (rawkode) [reply projects/rawkode-academy/image-service/package.json:7]: Yeah. `"prebuild": "just copy-wasm"` would work.

Comment (icepuma) [reply projects/rawkode-academy/image-service/package.json]: I have no real objection, so here we go :)

**Review by rawkode — APPROVED** [review]

**Pull request merged**